src/Endpoints/Daemons.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+namespace Cyberfusion\ClusterApi\Endpoints;
+
+use Cyberfusion\ClusterApi\Exceptions\RequestException;
+use Cyberfusion\ClusterApi\Models\Daemon;
+use Cyberfusion\ClusterApi\Request;
+use Cyberfusion\ClusterApi\Response;
+use Cyberfusion\ClusterApi\Support\ListFilter;
+
+class Daemons extends Endpoint
+{
+    /**
+     * @throws RequestException
+     */
+    public function list(?ListFilter $filter = null): Response
+    {
+        if (!$filter instanceof ListFilter) {
+            $filter = new ListFilter();
+        }
+
+        $request = (new Request())
+            ->setMethod(Request::METHOD_GET)
+            ->setUrl(sprintf('daemons?%s', $filter->toQuery()));
+
+        $response = $this
+            ->client
+            ->request($request);
+        if (!$response->isSuccess()) {
+            return $response;
+        }
+
+        return $response->setData([
+            'crons' => array_map(
+                fn (array $data) => (new Daemon())->fromArray($data),
+                $response->getData()
+            ),
+        ]);
+    }
+
+    /**
+     * @throws RequestException
+     */
+    public function get(int $id): Response
+    {
+        $request = (new Request())
+            ->setMethod(Request::METHOD_GET)
+            ->setUrl(sprintf('daemons/%d', $id));
+
+        $response = $this
+            ->client
+            ->request($request);
+        if (!$response->isSuccess()) {
+            return $response;
+        }
+
+        return $response->setData([
+            'daemon' => (new Daemon())->fromArray($response->getData()),
+        ]);
+    }
+
+    /**
+     * @throws RequestException
+     */
+    public function create(Daemon $daemon): Response
+    {
+        $this->validateRequired($daemon, 'create', [
+            'name',
+            'command',
+            'node_ids',
```

Comment:
Renamed to `nodes_ids` (it's the `ids` of multiple `nodes`, not the `ids` of a single `node`)